sdks/python/apache_beam/io/gcp/bigquery.py

```diff
@@ -2116,19 +2116,20 @@ class ReadFromBigQuery(PTransform):
     on GCS, and then reads from each produced file. File format is Avro by
     default.
 
+  .. warning::
+      DATETIME columns are not supported. They are currently parsed as strings
```

Comment:
could you please clarify why Datetime is not supported? `EXPORT` method is also using `fastavro` and supports `DATETIME` types

>  When using Avro exports, these fields will be exported as native Python
      types (datetime.date, datetime.datetime, datetime.datetime,
      and datetime.datetime respectively). Avro exports are recommended.

Reply:
I believe the issue is that `DATETIME` columns aren't translated to native Python types, not that they aren't supported at all (they are converted to strings rather than to native Python datetime types). This is the same as with Avro file-based exports from BigQuery today: https://cloud.google.com/bigquery/docs/exporting-data#avro_export_details

Reply:
@vachan-shetty can we update the docstring to be clearer here?

Reply:
Done